jbpm-human-task/jbpm-human-task-workitems/src/main/java/org/jbpm/services/task/wih/ExternalTaskEventListener.java

```diff
@@ -91,27 +94,17 @@ public void afterTaskStoppedEvent(Task ti) {
     }
 
     public void afterTaskCompletedEvent(TaskEvent event) {
-    	Task task = event.getTask();
+        Task task = event.getTask();
         long processInstanceId = task.getTaskData().getProcessInstanceId();
         if (processInstanceId <= 0) {
             return;
         }
-        RuntimeManager manager = getManager(task);
-        if (manager == null) {
-            throw new RuntimeException("No RuntimeManager registered with identifier: " + task.getTaskData().getDeploymentId());
-        }
-        RuntimeEngine runtime = manager.getRuntimeEngine(ProcessInstanceIdContext.get(processInstanceId));
-        KieSession session = runtime.getKieSession();
-        if (session != null) {
-            logger.debug(">> I've recieved an event for a known session (" + task.getTaskData().getProcessSessionId()+")");
-            processTaskState(task);
-        } else {
-            logger.error("EE: I've recieved an event but the session is not known by this handler ( "+task.getTaskData().getProcessSessionId()+")");
-        }
+        processTaskState(task);
```

Comment:
@ruivieira Not sure about this one, can you please explain why this is not needed anymore?

Reply:
This was removed in mistake, reverted.

Reply:
The removed code is still there. Can you also rebase on `master` because right know the PR contains also unrelated commits.

Reply:
@MarianMacik reverted now, thanks.